File: list/ListNode.java
package list;

public class ListNode {
	private Integer val;
	private ListNode next;

	public String toString(){
		return val.toString();
	}

	public ListNode(){
		this.val = 0;
		this.next = null;
	}

	public Integer getVal(){
		return this.val;
	}

	public void setVal(Integer val){
		this.val = val;
	}

	public ListNode getNext(){
		return this.next;
	}

	public void setNext(ListNode next){
		this.next = next;
	}
}

File: list/listNode.py
class Node:
    def __init__(self):
        self.val = 0
        self.next = None

    def __str__(self):
        result = ''
        if self.val == 0 and self.next == None:
            return result
        else:
            result = str(self.val)
            return result

    def getVal(self):
        return self.val

    def getNext(self):
        return self.next

    def setVal(self, val):
        self.val = val

    def setNext(self, next):
        self.next = next
